Tests/ModelsTests/CollectionTests.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+@testable import Models
+
+final class CollectionTests: XCTestCase {
+  func testAllCollections() {
+    XCTAssertEqual(Episode.Collection.all, Episode.Collection.all)
```

Comment:
I put this silly test in because without the laziness it will compile (using just computed vars) but it will crash at runtime. so just wanted a test to load up the collections to make sure all is fine